DOC: exmouth gulf (#74)
jbathmann/pyMANGA: web/content/en/docs/Sample_Model:_Exmouth_Gulf/model_results.md

> ---

Reviewer: s.o.

---
title: "The Model"
linkTitle: "The Model"
weight: 3
description:
---

<head>
<style type="text/css">
<!--
#vis {
  border: 1px solid black;
}
#Rahmen {
        border-width: 0.1em; 
        border-style: solid;
        text-align:right;
}
-->
</style>
</head>

> As already described in the <a href="/docs/sample_model_exmouth_gulf/exmouth_gulf/">introductory chapter</a>, there are numerous mangrove forests in the Exmouth Gulf. With the help of MANGA, the growth dynamics of the mangroves are reproduced in a model. The main focus is on the structure typical for mangrove forests and the dominance of individual mangrove species under different site conditions. In addition to the growth behavior of the mangroves, the salinity of the pore water - also as a sensitive parameter influencing growth - is considered in more detail. In this context the interacting influences of mangrove growth and gradients of salinity in pore water are also considered. This chapter describes the composition of the model, the basics of modeling with MANGA, and the results of the study.

Reviewer: I usually use porewater instead of "pore water"

Author: All replaced

# Model Domain

> A transect 185 m long and 10 m wide is represented in the model (<a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_1">Figure 1</a>).

Reviewer: ```suggestion
A 185 m long and 10 m wide transect is represented in the model (<a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_1">Figure 1</a>).
```

Author: Done

> Allometry data from a transect measurement, as well as measurement results from a long-term study, are used to evaluate the model results. The long-term study documents the effects of fertilization on the growth of 72 mangroves, half of them are located on the landward and half of them on the seaward side of the transect (see also <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_4">Figure 4</a>).

Reviewer: ```suggestion
Allometry data from a transect measurements, as well as measurement results from a long-term study, are used to evaluate the model results. The long-term study documents the effects of fertilization on the growth of 72 mangroves, half of them are located on the landward and half of them on the seaward side of the transect (see also <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_4">Figure 4</a>).
```

Author: Done

</figure><br>

The model shows the mangrove growth at the location in the Exmouth Gulf shown in the following figure.

<iframe src="https://www.google.com/maps/d/embed?mid=1EiX5yyZGJgVSu7pueUi5_jK160ndg0tG" width="640" height="480"></iframe>

<br>
<br>

Elevation profiles from the digital elevation model of the five transects (<a href="/en/docs/example-model_exmouth_gulf/results/#Figure_2">Figure 2 A</a>) were overlaid and the top of the terrain from the five signals (<a href="/en/docs/example_exmouth_gulf/results/#Figure_2">Figure 2 B</a>) was interpolated from the baseline of the signals. This DEM was used for descretization of the groundwater model (<a href="/en/docs/example_exmouth_gulf/results/#Figure_2">Figure 2 C</a>).

<figure>
<a name="Figure_2"></a>
<img src="/pictures/exmouth_gulf/dem.png" style="width:75%">
<figcaption><font size = "1"><i><b>Figure 2:</b> Elevation profiles along transect lines</i></font></figcaption>
</figure><br>

# Modeling

## Model variants

Mangrove growth was simulated using three different models (see also <a href="/docs/sample_model_exmouth_gulf/model_results/#Table_1">Table 1</a>). 

> In the model "<b>Model Without Feedback</b>" the dynamic changes in abiotic influences (tides, groundwater recharge and salinity of seawater) are included via boundary conditions. The influence of plant water extraction on pore water salinity was not modeled.

Reviewer: ```suggestion
In the model "<b>Model Without Feedback</b>" the dynamic changes in abiotic influences (tides, groundwater recharge and salinity of seawater) are included via boundary conditions. The influence of plant water extraction on pore water salinity was not accounted for.
```

Author: Done

<figure>
<a name="Figure_2"></a>
<img src="/pictures/exmouth_gulf/dis.png">
<figcaption><font size = "1"><i><b>Figure 2:</b> Spatial discretization of the groundwater model</i></font></figcaption>
</figure><p>

The mangroves extract soil water from the subsurface from a depth of 40 cm to 80 cm below the ground surface. <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_3">Figure 3</a> shows the model area (gray) and the area of water extraction by the mangroves (blue). Note the 50-fold vertical scaling.

<figure>
<a name="Figure_3"></a>
<img src="/pictures/exmouth_gulf/model_area_legend.png">
<figcaption><font size = "1"><i><b>Figure 3:</b> Area of water extraction by mangroves</i></font></figcaption>
</figure><p>

The groundwater model is discretized in time with a step length of one hour. The tidal range as a dynamic boundary condition is represented with the time series of the years 1991 to 1993, which is set in loops over the entire model runtime.

### Tree growth model

> Since each mangrove is represented as a single individual, a spatial discretization in proper sense does not exist. Temporally, the tree growth model is discretized with a time step length of half a year (1&nbsp;a&nbsp;=&nbsp;365.25&nbsp;d).

Reviewer: ```suggestion
Since each mangrove is represented as a single individual, there is no real spatial discretization. Temporally, the tree growth model is discretized with a time step length of half a year (1&nbsp;a&nbsp;=&nbsp;365.25&nbsp;d).
```

Author: Done

## Boundary conditions groundwater model

> The salinity of the seawater was set at 50 g/kg, the pore water at the landward end of the transect was assigned a salinity of 70 g/kg. The transpiration of the mangroves locally increases the salinity. The water level is determined in terms of hydrostatic pressure at the seaward and landward ends of the model area. In order to represent the tides, the lake-side water level was integrated into the model as a dynamic dirichlet boundary condition. The water level measurements of the Department of Transport of the Government of Western Australia served as data basis. The land-based water level is represented by a constant dirichlet boundary condition. The evparation of the trees is integrated as sinks in the area of the roots (see also <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_3">Figure 3</a>). Inflow in the form of precipitation is indirectly considered via salinity at the landward edge of the model area. The model boundaries not mentioned here are all defined as no flow boundary conditions. <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_1">Figure 1</a> shows a schematic diagram of this.

Reviewer: ```suggestion
The salinity of the seawater was set at 50 g/kg, the pore water at the landward end of the transect was assigned a salinity of 70 g/kg. The transpiration of the mangroves locally increases the salinity. The water level is determined in terms of hydrostatic pressure at the seaward and landward ends of the model area. In order to represent the tides, the lake-side water level was integrated into the model as a dynamic Dirichlet boundary condition. The water level measurements of the Department of Transport of the Government of Western Australia served as data basis. The land-based water level is represented by a constant Dirichlet boundary condition. Evaporation of the trees is integrated by sink terms in the area of the roots (see also <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_3">Figure 3</a>). Inflow in the form of precipitation is indirectly considered via salinity at the landward edge of the model area. The model boundaries which are not mentioned explicitly, are all defined as no flow boundary conditions. <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_1">Figure 1</a> shows a schematic diagram of this.
```

Author: Done

<th width="25%" style="text-align: left;">Rhizophora mangle </th>
</tr>
</thead>
<tbody>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>r</em><sub><em>R</em></sub></span></td>
<td style="text-align: left;">Root radius</td>
<td style="text-align: left;">0.25 m</td>
<td style="text-align: left;">0.25 m</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>r</em><sub><em>C</em></sub></span></td>
<td style="text-align: left;">Crown radius</td>
<td style="text-align: left;">0.3 m</td>
<td style="text-align: left;">0.3 m</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>r</em><sub><em>S</em></sub></span></td>
<td style="text-align: left;">Stem radius</td>
<td style="text-align: left;">0.01 m</td>
<td style="text-align: left;">0.01 m</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>h</em><sub><em>R</em></sub></span></td>
<td style="text-align: left;">Root depth</td>
<td style="text-align: left;">0.015 m</td>
<td style="text-align: left;">0.015 m</td>
</tr>
</tbody>
</table>

# Resource competition

> For representing the mangroves in the model area, it is necessary to establish a stable population, which means to reach quasi-stationary conditions. For this purpose, 30 mangroves are randomly positioned in the model area as seedlings. In each time step (length: half a year), 30 new mangroves are added, which are also randomly positioned in the model area. Due to the competition-based tree growth model, these new mangroves die more or less quickly. Thus, the probability that a young mangrove in the catchment area of an already older one dies again very quickly is very high. The reason for this is the above-ground competition, especially the lack of sunlight. Due to the concentration of salinity, caused by extraction of fresh water from the other mangroves, salt plumes are formed in the pore water. These provide worse growth conditions for the mangroves in downstream (especially for young mangroves). Different mangrove species have a higher or lower tolerance to high salt concentrations. In this research, the two species Avicennia marina ("gray mangrove") and Rhizophora mangle ("red mangrove") were studied in more detail.

Reviewer: ```suggestion
For representing the mangroves in the model area, it is necessary to establish a stable population, which means to reach quasi-stationary conditions. For this purpose, 30 mangroves are randomly positioned in the model area as seedlings. In each time step (length: half a year), 30 new mangroves are added, which are also randomly positioned in the model area. Due to the competition-based tree growth model, these new mangroves die more or less quickly. Thus, the probability that a young mangrove in the catchment area of an already older one dies again very quickly is very high. The reason for this is the above-ground competition, especially the lack of sunlight. Due to the concentration of salinity, caused by extraction of fresh water from the other mangroves, salt plumes are formed in the pore water. These lead to growth penalties for the mangroves located downstream (especially for young mangroves). Different mangrove species have varying tolerance to high salt concentrations. In this research, the two species Avicennia marina ("gray mangrove") and Rhizophora mangle ("red mangrove") were studied in more detail.
```

Author: Done

# Results

> In this research, two processes were viewed more closely with the help of the MANGA model. On the one hand, the development of typical structures in mangrove forests should be mapped, on the other hand, the growth behavior of the two mangrove species under different environmental conditions should be investigated. In the following the results of the research are briefly summarized.

Reviewer: ```suggestion
In this research, two processes were viewed more closely with the help of the MANGA model. On the one hand, the development of typical structures in mangrove forests is mapped, on the other hand, the growth behavior of the two mangrove species under different environmental conditions is investigated. In the following the results of the research are briefly summarized.
```

Author: Done

</p>
</figure>
<figcaption><font size = "1"><i><b>Visualisation 1:</b> Dynamic development of the mangrove population over the modeling period</i></font></figcaption>
<p>

<!--
<p>
<input type="button" value="click to go fullscreen" onclick="toggleFullScreen()">
</p>

<script type="text/javascript">
 /*<![CDATA[*/
function toggleFullScreen() {
  if ((document.fullScreenElement && document.fullScreenElement !== null) ||    
   (!document.mozFullScreen && !document.webkitIsFullScreen)) {
    if (document.documentElement.requestFullScreen) {  
      document.documentElement.requestFullScreen();  
    } else if (document.documentElement.mozRequestFullScreen) {  
      document.documentElement.mozRequestFullScreen();  
    } else if (document.documentElement.webkitRequestFullScreen) {  
      document.documentElement.webkitRequestFullScreen(Element.ALLOW_KEYBOARD_INPUT);  
    }  
  } else {  
    if (document.cancelFullScreen) {  
      document.cancelFullScreen();  
    } else if (document.mozCancelFullScreen) {  
      document.mozCancelFullScreen();  
    } else if (document.webkitCancelFullScreen) {  
      document.webkitCancelFullScreen();  
    }  
  }  
}

</script>
-->

> In the following video the model area was divided into ten sectors. The dynamic development of the mangrove population and the salt concentration in the bottom water as well as the biomass of the mangroves in the individual sectors are shown. Compared to the previous visualization, one main cause of the formation of the typical forest structure can be seen in this video, namely the concentration of salinity in the pore water in certain areas. The high correlation between salt concentration and biomass in the individual sectors can be seen already from a model runtime of 40 years. Already from 100 years, the developing structure typical for mangrove forests is recognizable.

Reviewer: ```suggestion
In the following video the model area was divided into ten sectors. The dynamic development of the mangrove population and the salt concentration in the bottom water as well as the biomass of the mangroves in the individual sectors are shown. Compared to the previous visualization, one main cause of the formation of the typical forest structure can be seen in this video, namely the concentration of salinity in the pore water in certain areas. The high correlation between salt concentration and biomass in the individual sectors can be seen already from a model runtime of 40 years. Already from 100 years, the structure of typical mangrove forests is recognizable.
```

Author: Done

<figure>
<a name="Figure_4"></a>
<img src="/pictures/exmouth_gulf/Figure_3.png">
<figcaption><font size = "1"><i><b>Figure 4:</b> Results</i></font></figcaption>
</figure><p>

> The "Treatment Averages" plotted in <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_4">Figure 4</a> are two areas where mangroves growing have been long term studied in more detail. A comparison of the results of these observations with the modeling results also shows a high degree of agreement.

Reviewer: ```suggestion
The "Treatment Averages" plotted in <a href="/docs/sample_model_exmouth_gulf/model_results/#Figure_4">Figure 4</a> are from mangroves that have been studied in more detail in long-term experiments. A comparison of the results of these observations with the modeling results also shows a high degree of agreement.
```

> In order to investigate the effects of taking into account the temporal dynamics of tides and plant water extraction on the salinity in the pore water, this effect was normalized using the following formula:

Reviewer: ```suggestion
In order to investigate the effects of the temporal dynamics of tides and plant water extraction on the salinity in the pore water, this effect was normalized using the following formula:
```

Author: Done

<figure>
<div align="center">
<img src="/pictures/exmouth_gulf/formel.png" width="50%">
</div>
</figure><p>

These relative effects are shown in the following <a href="/docs/sample_setup_exmouth_gulf/model_results/#Figure_5">Figure 5</a> for tree height and pore water salinity. A value of zero would mean that there is no difference in results between Full Model and the respective simplified model type. The larger the value becomes, the higher the deviation.

<figure>
<a name="Figure_5"></a>
<img src="/pictures/exmouth_gulf/Figure_3_2.png">
<figcaption><font size = "1"><i><b>Figure 5:</b> Relative impact of not considering tidal range ("Model Wihtout Tide") and plant water extraction ("Model Without Feedback").</i></font></figcaption>
</figure><p>

> Due to the greater effects of the tidal range in the area close to the see, the model "<b>Without Tide</b>" can only represent the tree heights and the pore water salinity here with a relatively large deviation compared to the "<b>Full Model</b>". However, with further moving towards the mainland, the water level fluctuations due to tides become smaller. Tree heights and salinities can be represented in this range (x > 75 m) with smaller relative deviations from the "<b>Full Model</b>".

Reviewer: ```suggestion
Due to the greater effects of the tidal range in the area close to the sea, the model "<b>Without Tide</b>" can only represent the tree heights and the pore water salinity here with a relatively large deviation compared to the "<b>Full Model</b>". However, with further inlands, the water level fluctuations due to tides become smaller. Tree heights and salinities can be represented in this range (x > 75 m) with smaller relative deviations from the "<b>Full Model</b>".
```

Author: Done

> The Model "<b>Without Feedback</b>" has problems mapping mangrove growth height as the "<b>Full Model</b>" does, especially in the middle to landward area (60 m < x < 165 m) of the transect. In this area the salinity of the pore water is concentrated by the plant water extraction, but this is not represented in this type of model.

Reviewer: ```suggestion
The Model "<b>Without Feedback</b>" fails to predict mangrove growth height, especially in the middle to landward area (60 m < x < 165 m) of the transect. In this area the salinity of the pore water is concentrated by the plant water extraction, but this is not represented in this type of model.
```

Author: Done

<figure>
<a name="Figure_7"></a>
<img src="/pictures/exmouth_gulf/Spezien_2.png" style="width:75%">
<figcaption><font size = "1"><i><b>Figure 7:</b> Dependence of species dominance on landward (x-axis) and seaward (y-axis) salinity.</i></font></figcaption>
</figure><p>

> When looking at the mixed forests, it is noticeable that real mixed populations are existing only in a few individual sections. In most areas a clear dominance of one species is formed. These sharp transitions between the individual dominance zones show that coexistence between the different species is only possible in areas of certain pore water salinities. The location of the boundaries and the change in species dominance d (slope of the curve) depend on the individual-specific parameters in the tree growth model. Note here that soil water salinity is also affected by the number of individuals per area and tree heights. These two parameters are in turn influenced by the same individual-specific parameters. 

Reviewer: ```suggestion
When looking at the mixed forests, real mixed populations are existing only in a few individual sections. In most areas a clear dominance of one species is expressed. These sharp transitions between the individual dominance zones show that coexistence between the different species is only possible in areas of certain pore water salinities. The location of the boundaries and the change in species dominance d (slope of the curve) depend on the individual-specific parameters in the tree growth model. Soil water salinity is also affected by the number of individuals per area and tree heights. These two parameters are in turn influenced by the same individual-specific parameters. 
```

Author: Done

Based on the results of the modeling, it must be assumed that a correct representation of mangrove growth with MANGA is only possible if the tidal range and the influences of water extraction of the mangroves from the subsurface are considered. Calibration of the plant parameters is not necessary for this purpose. Also not considered are heterogeneous hydrogeological properties of the subsurface, e.g. concerning hydraulic conductivity or porosity. 

The gradients of the salt concentration in the pore water caused by the plant water extraction have a significant effect on the growth dynamics of the mangrove population, especially in the landward area. Further, it can be concluded from the results that the influence of tides is a major influencing factor on the gradients of salinity concentration in bottom water. This influence is greatest at the seaward end of the transect. As the height of the tide decreases, or the duration of inundation decreases, the feedback between plant water and bottom water budgets takes on increasing importance in this process.

> Using the sensitivity analysis of the model with respect to species dominance, it was shown that species composition can be described by considering soil water balance and plant water withdrawal across the system boundary. By varying only two parameters (see <a href="/docs/sample_model_exmouth_gulf/model_results/#Table_3">Table 3</a>) that directly affect tree water uptake, typical zonation patterns in mixed mangrove forests could be reproduced. This was accomplished even with only crudely estimated plant-specific parameters for one of the two species. If the boundary conditions of the salt concentrations on the land- and seaward side are both chosen very high or very low, monoculture forests are formed. If a moderate mean value of the salinities is chosen, mixed forests of both species are formed. These show zones with clear dominance of one species, separated by sharp transitions. These transitions are shown to depend on the bottom water salinity. The species composition in the model agrees with the measured field data.

Reviewer: ```suggestion
Using the sensitivity analysis of the model with respect to species dominance, it was shown that species composition can be described by considering soil water balance and plant water withdrawal across the system boundary. By variation of just two parameters (see <a href="/docs/sample_model_exmouth_gulf/model_results/#Table_3">Table 3</a>) that directly affect tree water uptake, typical zonation patterns in mixed mangrove forests could be reproduced. This was accomplished even with roughly estimated plant-specific parameters for one of the two species. If the boundary conditions of the salt concentrations on the land- and seaward side are both chosen very high or very low, monoculture forests are formed. If a moderate mean value of the salinities is chosen, mixed forests of both species are formed. These show zones with clear dominance of one species, separated by sharp transitions. These transitions are shown to depend on the porewater salinity. The species composition in the model agrees with the measured field data.
```

Author: Done
